src/services/item/plugins/validation/service.ts

```diff
@@ -31,12 +32,19 @@ import { stripHtml } from './utils';
 
 export class ItemValidationService {
   itemService: ItemService;
+  private publishService: ItemPublishedService;
   fileService: FileService;
   imageClassifierApi?: string;
 
-  constructor(itemService: ItemService, fileService: FileService, imageClassifierApi?: string) {
+  constructor(
+    itemService: ItemService,
+    fileService: FileService,
+    publishService: ItemPublishedService,
```

Comment:
For now I inject the dependency manually, I will check if it is a small change to use a DI library for that.